enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/roles/Follower.java

```diff
@@ -81,17 +81,24 @@ public Outcome handle( RaftMessages.RaftMessage message, ReadableRaftState ctx,
         return message.dispatch( visitor( ctx, log ) );
     }
 
-    private abstract static class Handler implements RaftMessages.Handler<Outcome, IOException>
+    private static class Handler implements RaftMessages.Handler<Outcome,IOException>
     {
         protected final ReadableRaftState ctx;
         protected final Log log;
         protected final Outcome outcome;
+        private final PreVoteRequestHandler preVoteRequestHandlerDelegate;
+        private final PreVoteResponseHandler preVoteResponseHandler;
+        private final ElectionTimeoutHandler electionTimeoutHandler;
 
-        Handler( ReadableRaftState ctx, Log log )
+        Handler( PreVoteRequestHandler preVoteRequestHandlerDelegate, PreVoteResponseHandler preVoteResponseHandler,
+                ElectionTimeoutHandler electionTimeoutHandler, ReadableRaftState ctx, Log log )
         {
             this.ctx = ctx;
             this.log = log;
             this.outcome = new Outcome( FOLLOWER, ctx );
+            this.preVoteRequestHandlerDelegate = preVoteRequestHandlerDelegate;
```

Comment:
Is there a reason to single this one out with Delegate in the name? Hard for me to see without IDE-support, but can't really notice anything special through cursory browsing here.

Reply:
Probably an oversight from an earlier version.